cdap-cli/src/main/java/co/cask/cdap/cli/command/artifact/DescribeArtifactCommand.java

```diff
@@ -87,8 +88,9 @@ public String getPattern() {
 
   @Override
   public String getDescription() {
-    return String.format("Shows information about %s. If no scope is given, the user scope will be used. " +
-      "Includes information about application and plugin classes contained in the artifact.",
+    return String.format("Shows information about %s. If no scope is given, the artifact is looked up in all " +
```

Comment:
Do you think it's important? I thought when you say it's looked up, I didn't think that order matters. I can add it, but just thought it would make no difference.